extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -89,6 +89,15 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 		if (!this.currentBook) {
 			this.currentBook = book;
 		}
+		this._bookViewer = vscode.window.createTreeView(this.viewId, { showCollapseAll: true, treeDataProvider: this });
+		this._bookViewer.onDidChangeVisibility(e => {
+			if (e.visible) {
+				this.revealActiveDocumentInViewlet();
+			}
+		});
+		azdata.nb.onDidChangeActiveNotebookEditor(e => {
+			this.revealActiveDocumentInViewlet(e.document.uri, false);
```

Comment:
Just checking - this handles the case where you move to a different editor by unselecting the tree item in the viewlet right?

Reply:
Yeah, that works.